tests/integration/test_decision_worker.py

```diff
@@ -0,0 +1,153 @@
+"""
+Test worker process.
+"""
+
+import multiprocessing as mp
+import time
+
+from modules.cluster_estimation import cluster_estimation
+from modules.decision import decision_worker
+from modules.decision import decision
+from modules import drone_odometry_local
+from modules import odometry_and_time
+from utilities.workers import worker_controller
+from utilities.workers import queue_proxy_wrapper
+
+
+DISTANCE_SQUARED_THRESHOLD = 25  # unit of area
```

Comment:
This doesn't seem like area. It's used in `LandingPadTracking` and the comment there is: `# Landing pads within the square root of this distance are considered the same landing pad`

A comment like: `# Square root of this is metres` would work.